installing/installing_nutanix/nutanix-failure-domains.adoc

```diff
@@ -10,14 +10,54 @@ By default, the installation program installs control plane and compute machines
 
 A failure domain represents an additional Prism Element instance that is available to {product-title} machine pools during and after installation.
 
-include::modules/installation-nutanix-failure-domains-req.adoc[leveloffset=+1]
-
+[id="nutanix-failure-domains-install-method_{context}"]
 == Installation method and failure domain configuration
 
 The {product-title} installation method determines how and when you configure failure domains:
 
 * If you deploy using installer-provisioned infrastructure, you can configure failure domains in the installation configuration file before deploying the cluster. For more information, see xref:../../installing/installing_nutanix/installing-nutanix-installer-provisioned.adoc#installation-configuring-nutanix-failure-domains_installing-nutanix-installer-provisioned[Configuring failure domains].
 +
-You can also configure failure domains after the cluster is deployed. For more information about configuring failure domains post-installation, see xref:../../post_installation_configuration/adding-nutanix-failure-domains.adoc#adding-failure-domains-to-an-existing-nutanix-cluster[Adding failure domains to an existing Nutanix cluster].
+You can also configure failure domains after the cluster is deployed. For more information about configuring failure domains post-installation, see xref:../../installing/installing_nutanix/nutanix-failure-domains.adoc#nutanix-failure-domains-adding-to-existing-cluster_nutanix-failure-domains[Adding failure domains to an existing Nutanix cluster].
 
 * If you deploy using infrastructure that you manage (user-provisioned infrastructure) no additional configuration is required. After the cluster is deployed, you can manually distribute control plane and compute machines across failure domains.
+
+[id="nutanix-failure-domains-adding-to-existing-cluster_{context}"]
+== Adding failure domains to an existing Nutanix cluster
+
+By default, the installation program installs control plane and compute machines into a single Nutanix Prism Element (cluster). After an {product-title} cluster is deployed, you can improve its fault tolerance by adding additional Prism Element instances to the deployment using failure domains.
+
+A failure domain represents a single Prism Element instance to which:
+
+* New control plane and compute machines can be deployed.
+* Existing control plane and compute machines can be distributed.
```

Comment:
IBM SG says avoid continuing a sentence into list items. This is a poorly written example suggestion, but e.g.:
```suggestion
A failure domain represents a single Prism Element instance to which the following apply:

* New control plane and compute machines can be deployed
* Existing control plane and compute machines can be distributed
```
And note lack of punctuation if the list items end up being fragments.